### PR TITLE
feat: group tickets added to tests

### PR DESCRIPTION
**Pull request by RobCatton**
# Description

-   Group tests included in UI automation tests.

# Testing instructions

-   Add testing instructions if required

# Type of change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New test for full journey (non-breaking change)

# Checklist:

-   [ ] Able to run pr locally
-   [ ] Followed test acceptance criteria
-   [ ] My code follows the style guidelines of this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have made corresponding changes to the documentation if applicable

### CODE REVIEW

File: src/main/java/uk/co/tfn/HelperMethods.java [hunk 1]
```diff
@@ -433,33 +438,97 @@ public void completeUserDetailsPage() {
...
-                        .waitForElement(String.format("passenger-type-%s", String.valueOf(randomUserType)));
+        if(randomSelector == 1){
+            // Click Any and continue
+            WebElement element = this.waitForElement("passenger-type-anyone");
+            if (this.browser.equals("ie")) {
+                JavascriptExecutor executor = (JavascriptExecutor) this.driver;
+                executor.executeScript("arguments[0].click();", element);
+            } else {
+                element.click();
+            }
```
Comment (dannydaviesIW) [inline src/main/java/uk/co/tfn/HelperMethods.java:454]: Looks like we use this chunk of code a fair few times - would it be worth splitting it out into it's own function?

File: src/main/java/uk/co/tfn/HelperMethods.java [hunk 2]
```diff
@@ -433,33 +438,97 @@ public void completeUserDetailsPage() {
...
+                executor.executeScript("arguments[0].click();", chosenPassenger);
+            } else {
+                chosenPassenger.click();
+            }
+            this.continueButtonClick();
+            this.waitForPageToLoad();
+            this.completeUserDetailsPage(false, 0);
+        }
+    }
+
```
Comment (dannydaviesIW) [inline src/main/java/uk/co/tfn/HelperMethods.java:491]: Should the below methods live in `StepMethods.java `instead?

File: src/main/java/uk/co/tfn/HelperMethods.java (hunk 1 above)
Comment (RobCatton) [reply src/main/java/uk/co/tfn/HelperMethods.java:454]: Done!

File: src/main/java/uk/co/tfn/HelperMethods.java (hunk 2 above)
Comment (RobCatton) [reply src/main/java/uk/co/tfn/HelperMethods.java:491]: Think its ok where it is, its in an else block so we only want it to run in a certain scenario.

**Review by dannydaviesIW — APPROVED** [review]

**Pull request merged**